react/features/base/config/reducer.js

```diff
@@ -320,6 +320,18 @@ function _translateLegacyConfig(oldValue: Object) {
         newValue.e2ee.e2eeLabels = oldValue.e2eeLabels;
     }
 
+    if (oldValue.defaultLocalDisplayName === undefined
```

Comment:
Note that since you commented out the defaults the code needs to provide a default when both are undefined.

Reply:
Replaced these with a check based on the new config obj, if a value is added we take that, if not, we take the default.

Reply:
Reverted.